README.md

```diff
@@ -23,15 +23,15 @@ Currently, the following adapters are supported:
 ### Cloning via dbt Package Hub
   
 Check [dbt Hub](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) for the latest installation instructions, or [read the docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
-### Additional setup for Databricks/Spark
+### Additional setup for Databricks/Spark/DuckDB/Redshift
 
 In your `dbt_project.yml`, add the following config:
 ```yml
 # dbt_project.yml
 
 dispatch:
-  - macro_namespace: dbt_utils
-    search_order: ['dbt_project_evaluator', 'spark_utils', 'dbt_utils']
+  - macro_namespace: dbt
+    search_order: ['dbt_project_evaluator', 'dbt']
 ```
 
 This is required because the project currently provides limited support for arrays macros for Databricks/Spark which is not part of `spark_utils` yet.
```

Comment:
Is this still the reason? I saw you added DuckDB and Redshift to the Additional Setup header

Reply:
No, that reason should be updated. All the array macros are in the `dbt` namespace, and we have a couple shims in place. I can update this!